system-variables.md

```diff
@@ -1,129 +1,690 @@
 ---
-title: The System Variables
-summary: Learn how to use the system variables in TiDB.
+title: System Variables
+summary: Use system variables to optimize performance or alter running behavior.
 category: reference
-aliases: ['/docs/dev/system-variables/','/docs/dev/reference/configuration/tidb-server/mysql-variables/']
+aliases: ['/docs/dev/tidb-specific-system-variables/','/docs/dev/reference/configuration/tidb-server/tidb-specific-variables/', '/tidb/dev/tidb-specific-system-variables']
```

Comment:
```suggestion
aliases: ['/docs/dev/system-variables/','/docs/dev/reference/configuration/tidb-server/mysql-variables/','/tidb/dev/tidb-specific-system-variables']
```
@nullnotnil Two notes for the aliases:

1. The existing aliases can be left alone. They are meant to redirect old URLs to this system-variables.md document.
2. Because tidb-specific-system-variables.md is now integrated into system-variables.md, so add an alias '/tidb/dev/tidb-specific-system-variables'.

Reply:
Because it is merging docs, I have to add the aliases from tidb-specific as well. So the new list is:
```
aliases: ['/docs/dev/system-variables/','/docs/dev/reference/configuration/tidb-server/mysql-variables/', '/docs/dev/tidb-specific-system-variables/','/docs/dev/reference/configuration/tidb-server/tidb-specific-variables/']
```

Reply:
You're right. There should be 4 aliases now. Please apply it~